new/LIT7098QeneAlam.xml

```diff
@@ -0,0 +1,97 @@
+                    <bibl><ptr target="bm:Guidi1900Qene"/></bibl>. 
+                    It is attributed to <persName ref="PRS14579WaldaGabreel"/> in the manuscript.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+                <language ident="sv">Swedish</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2024-10-11">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7098QeneAlam" passive="UppQ1524o"/>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7098QeneAlam" passive="PRS14579WaldaGabreel"/>
+                </listRelation>
+            </div>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="UppQ1524o"/> on fols 1r, 2r.
+                    Division into verses is indicated with ፤ .
+                    The end of the poem is indicated with ። .
+                    The first line was originally laid out as four lines in the manuscript, each ending with ፤ .
+                However, later this was noted and the lines were marked as constituting a single line by a speaker of
+                Swedish (who noted down <foreign xml:lang="sv">"1 rad"</foreign>, "one line").
+                Judging from the rhyme, lines 4 and 5 also appear to have been divided—each into two lines—but this has not
+                been noted in the manuscript.</note>
+                <ab>
+                    <l n="1">ዓለም፡ እስከ፡ ጊዜሃ<choice><sic>፤</sic><corr resp="JK">፡</corr></choice>
+                        በሐብለ፡ ዝንጋኤ፡ ትስሕበኒ<choice><sic>፤</sic><corr resp="JK">፡</corr></choice>
+                        ወታወርደኒ፡ ካእበ<choice><sic>፤</sic><corr resp="JK">፡</corr></choice> 
+                                <del rend="overUnderlined">ሐ</del><add place="above">ኀ</add>በ፡ ዘቀዳሚ፡ መርገም፤</l>
+                    <l n="2">ለዘተወለድነ፡ በሥጋ፡ ወደም፤</l><pb n="2r"/>
+                    <l n="3">ለሕልፈት፡ ዛቲ፡ ዓለም፤</l>
+                    <l n="4">ፍ<choice><sic>ደ</sic><corr resp="JK">ዳ</corr></choice>፡ 
+                        ወይን፡ ይስተይ<choice><sic>፤</sic><corr resp="JK">፡</corr></choice> 
+                        ማየ፡ ዝናም፤
+                    </l>
+                    <l n="5">ወህየንተ፡ ልብስ<choice><sic>፤</sic><corr resp="JK">፡</corr></choice> 
+                        ይትከደን፡ ቈጽለ፡ አእዋም፤
+                    </l>
+                    <l n="6">እስመ፡ ዛቲ፡ ጽላሎት፡ ወሕልም(ት)።</l>
```

Comment:
I think, the brackets are clearly by a later hand. They should be seen as a way to delete the final ት to bring the verse in accordance with the rhyme.

Reply:
Yes, but how to encode it? Check if you think what I did now might work

Reply:
@CarstenHoffmannMarburg , if you think this works, maybe this file can be approved?